book/whats_new/aug-18/swift-5.0-migration.md

```diff
@@ -0,0 +1,109 @@
+    * Inside each podspec file, make sure that the s.xcconfig.'SWIFT_VERSION' value is set to 5.0.
+    * Update all dependencies to support swift 5.0 specs.
+6. Update podfile structure according to the following steps:
+    * dependencies to support swift 5.0 specs.
+    * Use the new cocoapods CDN source:
+        * Remove `source 'git@github.com:CocoaPods/Specs.git'`.
+        * Add the new CDN `source  'https://cdn.cocoapods.org/'` (If you have private specs, add the new source below all of the other sources).
+        * Run `pod deintegrate` to deintegrate CocoaPods from your project. The move to the latest CDN source may cause problems, therefore, it’s recommended to clean all cocoapods related files.
+        * Run `pod install`.
+7. Run pod update command and open the project with Xcode 10.2.1.
+8. In side Xcode, in each of your TARGETS, change the “swift language version” to swift 5.
+9. Build the framework in Xcode and make sure it’s working without errors.
+10. Increment the plist framework version to a major change version notion.
+11. Push your code with GIT and make sure circle-ci build process ended successfully.
+12. Verify that a new podspec file was created inside our private [CocoaPods-Private](https://github.com/applicaster/CocoaPods-Private) or public [CocoaPods](https://github.com/applicaster/CocoaPods) repositories.
+13. Create a new manifest file that correlates to the same version number in the framework plist and upload to Zapp.
+
+## 3rd Party Plugins Migration
+
+The following instructions are for 3th party developers that are not using the applicaster closed frameworks logic.
+
+1. Install cocoapods version 1.7.5 using the following command: 
+    ```
+    sudo gem install cocoapods
+    ```
+2. Update podspec dependencies to support swift 5.0.
+    * Inside each podspec file, make sure that the s.xcconfig.'SWIFT_VERSION' value is set to 5.0.
+    * Update all dependencies to support swift 5.0 specs.
+3. Increment the podspec to a major change version notion.
+4. Create an empty xcode test project with xocde 10.2.1 and a new podfile that points to your upgraded podspec file.
+5. Update podfile structure (if you already have a test project)
+    * Move dependencies to support swift 5.0 specs
+    * Use the new cocoapods CDN source and do the following:
+        * Remove source `'git@github.com:CocoaPods/Specs.git'`.
+        * Add the new CDN `source ’https://cdn.cocoapods.org/'`  (If you have private specs, add the new source below all of the other sources).
+        * Run pod deintegrate to deintegrate CocoaPods from your project. The move to the latest CDN source may cause problems, therefore, it’s recommended to clean all cocoapods related files.
+        * Run `pod install`.
+6. Update the podfile dependencies to support swift 5.0 specs.
+7. In side xcode, in each of your TARGETS, change the “swift language version” to swift 5.
```

Comment:
same comment - this should be already "right" after pod install / update and shouldn't be manually override in the project

Reply:
done